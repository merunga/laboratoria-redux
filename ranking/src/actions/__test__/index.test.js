import * as actions from '../'

describe('actions', () => {
  it('crea action para upvote con id', () => {
    expect(actions.upVote(1)).toMatchSnapshot()
  })
  it('crea action para downVote con id', () => {
    expect(actions.upVote(2)).toMatchSnapshot()
  })
  it('crea action para prev', () => {
    expect(actions.prev()).toMatchSnapshot()
  })
  it('crea action para next', () => {
    expect(actions.next()).toMatchSnapshot()
  })
})