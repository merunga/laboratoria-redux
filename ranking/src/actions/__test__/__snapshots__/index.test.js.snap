// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions crea action para downVote con id 1`] = `
Object {
  "id": 2,
  "type": "UP_VOTE",
}
`;

exports[`actions crea action para next 1`] = `
Object {
  "canciones": Array [],
  "type": "NEXT",
}
`;

exports[`actions crea action para prev 1`] = `
Object {
  "canciones": Array [],
  "type": "PREV",
}
`;

exports[`actions crea action para select con id 1`] = `
Object {
  "id": 5,
  "type": "SELECT",
}
`;

exports[`actions crea action para upvote con id 1`] = `
Object {
  "id": 1,
  "type": "UP_VOTE",
}
`;
